.github/ISSUE_TEMPLATE/BUG_REPORT.md

```diff
@@ -0,0 +1,21 @@
+---
+name: Bug report
+about: Create a report to fix a bug
+---
+
+### Description
+<!-- Briefly describe the issue you're experiencing -->
+
+### Steps to reproduce
+<!-- what you were trying to do and what happened instead. -->
+1.
+1.
+
+### Which pipeline, environment and application did you use?
+<!-- Lumint / Currency Management Application -->
+<!-- Production / Staging / ReviewApp -->
+<!-- Production / Parallel / Demo / ... -->
+
+### Supporting data
+<!-- Do you have screenshots showing the problem? -->
```

Comment:
Maybe add something rel. to attach the filesif the error was uploading one, like a csv.